functions/_load_sushi.fish

```diff
@@ -69,7 +69,7 @@ end
 
 # Test whether this is a terraform directory by finding .tf files
 function terraform::directory
-	command find . -name '*.tf' >/dev/null ^/dev/null -maxdepth 0
+	command find . -name '*.tf' >/dev/null 2>/dev/null -maxdepth 0
```

Comment:
To be consistent, why not use "2>&1" instead ?

Reply:
Thanks for the suggestion!